docs/general/wallets.md

```diff
@@ -0,0 +1,155 @@
+---
+id: wallets
+title: Wallets
+sidebar_label: Wallets
+description: An overview of wallets in Polkadot ecosystem.
+keywords: [wallets, account, signer, supported wallets, balance, token]
+slug: ../wallets
+---
+
+Navigating the space of wallets can be overwhelming, if you are new to blockchain technology. In
+general, an account on a typical blockchain network is a public-private key pair. Access to a
+private key gives full access to all the allowed transactions on that blockchain account. It is
+important to keep the private key secure.
+
+:::tip Not your keys, not your tokens!
+
+With custodial wallets (like accounts central exchanges), another party controls your private keys.
+With non-custodial wallets, only you have access to your account's private key.
+
+:::
+
+To realize the multichain vision of Polkadot, it is important to have non-custodial wallets that
+make it convenient to interact with multiple blockchains in Polkadot ecosystem.
+
+## Non-Custodial Wallets
+
+:::caution
+
+You should use your own due diligence in researching and using the wallets listed below. For any
+issues related to these wallet, reach out to their support teams directly.
+
+:::
+
+Typically, the account keys are either stored and accessed through a browser extension or a
+smartphone app (which are considered as a hot wallets as they are exposed online) or an airgapped
+device or a hardware wallet (which considered as a cold wallets as they are offline).
+
+### Browser Wallets
+
+It is possible to use your browser to interact with your blockchain account through the wallets
+listed below. Typically, these wallets use the browser extension as a keystore for the blockchain
+accounts. Some of the browser extensions also allow interacting with airgapped devices and hardware
+wallets in a secure way.
```

Comment:
"Browser extension wallets at bare minimum act as key storage and management solution, allowing you to use your accounts with apps in the Web 3.0 space. The wallets listed below offer functionality beyond that, allowing the featured actions to be performed directly through the extension. Some of them also allow interaction with air-gapped wallets and hardware devices"

Reply:
Added the suggestion